Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -126,6 +128,60 @@ public void TestDependencyCollectionEventSource()
 
         [TestMethod]
         [Timeout(5000)]
+        public void TestDependencyCollectionEventSource404()
+        {
+            using (var module = new DependencyTrackingTelemetryModule())
+            {
+                module.DisableDiagnosticSourceInstrumentation = true;
+                module.ProfileQueryEndpoint = FakeProfileApiEndpoint;
+                module.Initialize(this.config);
+                Assert.IsFalse(DependencyTableStore.IsDesktopHttpDiagnosticSourceActivated);
+
+                var url = new Uri("http://google.com/404");
+                HttpClient httpClient = new HttpClient();
+                httpClient.GetStringAsync(url).ContinueWith(t => { }).Wait();
+
+                var item = this.sentTelemetry.Single();
+                Assert.AreEqual(url, item.Data);
+                Assert.AreEqual(url.Host, item.Target);
+                Assert.AreEqual(url.AbsolutePath, item.Name);
+                Assert.IsTrue(item.Duration > TimeSpan.FromMilliseconds(0), "Duration has to be positive");
+                Assert.AreEqual(RemoteDependencyConstants.HTTP, item.Type, "HttpAny has to be dependency kind as it includes http and azure calls");
+                Assert.IsTrue(
+                    DateTime.UtcNow.Subtract(item.Timestamp.UtcDateTime).TotalMilliseconds <
+                    TimeSpan.FromMinutes(1).TotalMilliseconds,
+                    "timestamp < now");
+                Assert.IsTrue(
+                    item.Timestamp.Subtract(DateTime.UtcNow).TotalMilliseconds >
+                    -TimeSpan.FromMinutes(1).TotalMilliseconds,
```

Comment:
The test timeout is 5 seconds but we are asserting that the duration is less than one minute?
If the test will fail after 5 seconds shouldn't this be checking for only 5 seconds worth of difference or less?